locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -938,7 +1058,11 @@ public void apply(TemplateOptions t, ConfigBag props, Object v) {
                     public void apply(TemplateOptions t, ConfigBag props, Object v) {
                         if (t instanceof EC2TemplateOptions) {
                             if (v==null) return;
-                            ((EC2TemplateOptions)t).userData(v.toString().getBytes());
+                            String data = v.toString();
+                            if (!(data.startsWith("<script>") || data.startsWith("<powershell>"))) {
+                                data = "<script> " + data + " </script>";
```

Comment:
This is documented in the AWS documentation here: http://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/UsingConfig_WinAMI.html#user-data-execution